### PR TITLE
Issue #12615: removes unnecessary code for EmptyBlock

### PR DESCRIPTION
**Pull request by rnveach**
Issue #12615

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java [hunk 1]
```diff
@@ -260,8 +260,7 @@ public void visitToken(DetailAST ast) {
                 }
                 if (emptyBlock) {
                     log(leftCurly,
-                        MSG_KEY_BLOCK_NO_STATEMENT,
-                        ast.getText());
+                        MSG_KEY_BLOCK_NO_STATEMENT);
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java:263]: https://github.com/checkstyle/checkstyle/search?q=block.noStatement

This message has no parameters, so the text given to it is not necessary.

**Comment by rnveach** [comment]
No regression is needed.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java (hunk 1 above)
Comment (nrmancuso) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java:263]: Good find, looks like there might be others like this:

```
➜  checkstyle git:(move-travis-tasks) ✗ ag -Q "removed call to com/puppycrawl/tools/checkstyle/api/DetailAST::getText"
config/pitest-suppressions/pitest-coding-1-suppressions.xml
89:    <description>removed call to com/puppycrawl/tools/checkstyle/api/DetailAST::getText</description>
143:    <description>removed call to com/puppycrawl/tools/checkstyle/api/DetailAST::getText</description>

config/pitest-suppressions/pitest-javadoc-suppressions.xml
323:    <description>removed call to com/puppycrawl/tools/checkstyle/api/DetailAST::getText</description>

config/pitest-suppressions/pitest-blocks-suppressions.xml
8:    <description>removed call to com/puppycrawl/tools/checkstyle/api/DetailAST::getText</description>


```

Comment (rnveach) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java:263]: I am going group by group for now. I am skipping over common areas and more focusing on checks.

**Review by romani — APPROVED** [review]

**Comment by romani** [comment]
Please fix conflict

**Comment by rnveach** [comment]
Done.

**Pull request merged**